book_maker/cli.py

```diff
@@ -187,12 +187,7 @@ def main():
                 "OpenAI API key not provided, please google how to obtain it"
             )
     elif options.model == "caiyun":
-        OPENAI_API_KEY = (
-            options.openai_key
-            or env.get(
-                "BBM_CAIYUN_API_KEY"
-            )
-        )
+        OPENAI_API_KEY = options.openai_key or env.get("BBM_CAIYUN_API_KEY")
```

Comment:
do not call it openai_key let us make a new arg

Reply:
I'm sorry. Can I change it to CAIYUN_API_KEY??
like this:
```python
if options.model in ["gpt3", "chatgptapi"]:
    ...
    API_KEY = OPENAI_API_KEY
elif options.model == "caiyun":
    API_KEY = options.openai_key or env.get("BBM_CAIYUN_API_KEY")
    
e = book_loader(
        options.book_name,
        translate_model,
        API_KEY,
        options.resume,
        language=language,
        model_api_base=model_api_base,
        is_test=options.test,
        test_num=options.test_num,
        translate_tags=options.translate_tags,
        allow_navigable_strings=options.allow_navigable_strings,
        prompt_config=parse_prompt_arg(options.prompt_arg),
        batch_size=options.batch_size,
    )
```